pkg/oc/admin/router/router.go

```diff
@@ -308,6 +311,7 @@ func NewCmdRouter(f *clientcmd.Factory, parentName, name string, out, errout io.
 	cmd.Flags().StringVar(&cfg.Ciphers, "ciphers", cfg.Ciphers, "Specifies the cipher suites to use. You can choose a predefined cipher set ('modern', 'intermediate', or 'old') or specify exact cipher suites by passing a : separated list. Not supported for F5.")
 	cmd.Flags().BoolVar(&cfg.StrictSNI, "strict-sni", cfg.StrictSNI, "Use strict-sni bind processing (do not use default cert). Not supported for F5.")
 	cmd.Flags().BoolVar(&cfg.Local, "local", cfg.Local, "If true, do not contact the apiserver")
+	cmd.Flags().StringVar(&cfg.Threads, "threads", cfg.Threads, "Specifies the amount of threads for the haproxy router.")
```

Comment:
Specifies the number of threads... (number instead of amount)

Reply:
Use cmd.Flags().Int32Var() instead of cmd.Flags().StringVar() then you don't need to do integer validations for this param.

Reply:
Is there a max thread limit? (very high thread count does not seem practical)

Reply:
i haven't tested the thread limit, i'll check that